Makefile

```diff
@@ -81,37 +90,37 @@ check-all: black-ci pylint mypy check-migrations pytest pytest-integration
 
 black-ci: compose-build
 	@echo "Running black checks..."
-	@docker-compose run $(IMAGE_NAME) \
+	@$(RUN_NO_DEPS) \
 		black --check src/ \
 		|| (echo "Error running 'black --check', please run 'make black' to format your code!"; exit 1)
 
 check-migrations: compose-build
 	@echo "Check if there are unrun migrations..."
-	@docker-compose run --rm $(IMAGE_NAME) \
+	@$(RUN) \
 	python -c "\
 	from fidesops.db.database import check_missing_migrations; \
 	from fidesops.core.config import config; \
 	check_missing_migrations(config.database.SQLALCHEMY_DATABASE_URI);"
 
 pylint: compose-build
 	@echo "Running pylint checks..."
-	@docker-compose run $(IMAGE_NAME) \
+	@$(RUN_NO_DEPS) \
 		pylint src/
 
 mypy: compose-build
 	@echo "Running mypy checks..."
-	@docker-compose run $(IMAGE_NAME) \
+	@$(RUN_NO_DEPS) \
 		mypy --ignore-missing-imports src/
 
 pytest: compose-build
 	@echo "Running pytest unit tests..."
-	@docker-compose run $(IMAGE_NAME) \
+	@$(RUN) \
 		pytest $(pytestpath) -m "not integration and not integration_external and not integration_saas"
 
 pytest-integration:
```

Comment:
OK, so catching up here, it looks like tests are failing for mariadb because we're trying to truncate tables that don't yet exist? If you remove the truncation bit, it will still fail because we're trying to populate mariadb tables that don't yet exist.

This file creates the tables `data/sql/mariadb_example_schema.sql, which should happen when we bring the mariadb container up.  I wonder if the data volume already exists somehow @ThomasLaPiana

Reply:
It's worth noting that `make clean` won't remove volumes right now, to do that you need to run `docker volume rm $(docker volume ls -q)`

Reply:
yeah but this test failure is happening in the CI checks, which is weird. Makes me think I just messed up a path or something?